### ISSUE TITLE
Proxy docs outdated, should point to fluentd instead

### ISSUE THREAD

**Issue by Sheeri** [OpenLineage/docs#293]
https://github.com/OpenLineage/docs/blob/main/docs/development/ol-proxy.md mentions the old proxy.

**Comment by dolfinus** [comment]
Do you mean something like this?
```ini
[INPUT]
    Name http
    Port 9200
    Mem_Buf_Limit 50MB
    successful_response_code 201
    success_header Content-Type application/json

[OUTPUT]
    Name http
    Match *
    host marquez-host
    port 5000
    uri /api/v1/lineage
```

**Comment by Sheeri** [comment]
No, I mean that https://github.com/OpenLineage/docs/blob/main/docs/development/ol-proxy.md contains instructions for how to build a proxy. At least some of this is outdated, for example:

_Not only the OpenLineage proxy is useful in receiving the monitoring the OpenLineage events, it can also be used to relay the events to other endpoints. Please see the [example](https://github.com/OpenLineage/OpenLineage/blob/main/proxy/proxy.example.yml) of how to set the proxy to relay the events via Kafka topic or HTTP endpoint._

Relaying events to endpoints is generally accepted to be best done by fluentd. I'm not sure if a proxy is still favorable for receiving/monitoring OpenLineage events in general, but if so, then most of the page should be rewritten.

**Comment by dolfinus** [comment]
My question was about fluentd configuration which should be compatible with OpenLineage clients. In the previous comment I've added an example of config for fluent-bit, which is quite similar to fluentd.

**Comment by Sheeri** [comment]
Right! Because fluentd is preferred as a proxy, what purpose does this "ol-proxy" serve?

**Comment by dolfinus** [comment]
Whell, Readme of fluentd plugin says:
> Fluentd support is experimental and could be changed or removed in a future release.
